### PR TITLE
Fix: Japanese doc

### PR DESCRIPTION
**Pull request by eltociear**
Add translate

### CODE REVIEW

File: ja/orm/retrieving-data-and-resultsets.rst [hunk 1]
```diff
@@ -368,7 +368,7 @@ fineder メソッドは、あなたが作成したい finder の名前が ``Foo`
     }
 
     // コントローラーやテーブルのメソッド内で
-    // Prior to 3.6 use TableRegistry::get('Articles')
+    // 3.6 より前は、 TableRegistry::get('Articles') を使用
```
Comment (okinaka) [inline ja/orm/retrieving-data-and-resultsets.rst:371]: This comment is for 3.x. Please change direction to 3.x branch.

Comment (okinaka) [reply ja/orm/retrieving-data-and-resultsets.rst:371]: Or, please try translating [here](https://github.com/cakephp/docs/blob/4.x/en/orm/retrieving-data-and-resultsets.rst).

Comment (othercorey) [reply ja/orm/retrieving-data-and-resultsets.rst:371]: Please remove the "prior to 3.x" code in 4.x docs.

Comment (eltociear) [reply ja/orm/retrieving-data-and-resultsets.rst]: @okinaka @othercorey Thank you for reviewing and commenting.
I did add commit with remove description of 3.x.

**Pull request merged**